client/state/sites/actions.js

```diff
@@ -90,6 +91,7 @@ export function requestSites() {
 				site_activity: 'active',
 				fields: SITE_REQUEST_FIELDS,
 				options: SITE_REQUEST_OPTIONS,
+				filters: config.isEnabled( 'jetpack-cloud' ) ? 'atomic,jetpack' : '',
```

Comment:
Do we want atomic here?
My answer right now is no.

Reply:
I would agree. For all intents and purposes, Atomic sites are dotcom sites in terms of UX so we should do our best to not fork their experience. That means avoiding bringing them into the Cloud but also not list Atomic sites inside JP Cloud, as I think is what's happening here.